extensions/amp-list/0.1/amp-list.css

```diff
@@ -41,16 +41,25 @@ amp-list[load-more] [load-more-button].i-amphtml-default-ui,
   font-size: 14px;
   text-transform: uppercase;
   letter-spacing: .1em;
-
   color: rgb(51,51,51);
   text-align: center;
 }
 
-[load-more-button].i-amphtml-list-load-more-overflow {
+amp-list[load-more] [load-more-loading],
+amp-list[load-more] [load-more-failed],
+amp-list[load-more] [load-more-button] {
   position: absolute;
   bottom: 0;
 }
 
+[load-more] div[role="list"] {
```

Comment:
you can just _not_ call `applyFillContent` when it is `load-more`.
Honestly I have no idea why we ever made the container be `fill`, it has been the case since day one and risky to change for default case now but makes no sense to me.

Reply:
if we don't make `applyFillContent` to he container, there is no longer a need to apply the bottom-margin and fixes a bug where if currently height of list is longer than height of items, show more appears below the list instead of below the items.